test/multiverse/lib/multiverse/output_collector.rb

```diff
@@ -71,6 +72,27 @@ def self.failing_output_footer
         red("*" * 80)]
     end
 
+    # Saves the failing out put to the working directory of the container
+    # where it is later read and output as annotations of the github workflow
+    def self.save_output_to_error_file(lines)
+      @output_lock.synchronize do
+        filepath = ENV["GITHUB_WORKSPACE"]
+        output_file = File.join(filepath, "errors.txt")
+
+        existing_lines = []
+        if File.exist?(output_file)
```

Comment:
The later write to `output_file` should be under this conditional too, right?

Reply:
I'm not quite sure what you're seeing here.  `existing_lines` is meant to be empty unless the file has previously been written.  

1. The first call to this method expects the file to not exist and thus `existing_lines` is `[]` which end up writing nothing for `existing_lines` since it's empty. `puts([])` **=> nil**
2. subsequent calls will populate `existing_lines` with the file's content and then both are written out to file.

Reply:
Ah, thanks for clarifying, never mind!